packages/docs/src/components/doc/Accessibility.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+  <div>
+    <doc-heading>Generic.Pages.accessibility</doc-heading>
+    <doc-text>{{ `${namespace}.${page}.accessibility.desc` }}</doc-text>
+  </div>
+</template>
+
+<script>
+  // Utilities
+  import {
+    mapGetters,
+  } from 'vuex'
+
+  export default {
+    props: {
+      value: {
+        type: [Object, String],
+        default: undefined,
+      },
+    },
+
+    computed: {
+      ...mapGetters('documentation', [
+        'namespace',
+        'page',
+      ]),
+      internalValue () {
```

Comment:
This doesn't appear to be used.

Reply:
This whole component or just some of it? You won't find a direct reference to the component because it [gets loaded dynamically from here](https://github.com/vuetifyjs/vuetify/blob/master/packages/docs/src/components/core/Page.vue#L24). The component type is [specified in the `Divider.json` flie](https://github.com/morphatic/vuetify/blob/v-divider-docs-updates/packages/docs/src/lang/en/components/Dividers.json#L29).

Reply:
This component is basically a modified and simplified copy of [the `Usage` component](https://github.com/morphatic/vuetify/blob/v-divider-docs-updates/packages/docs/src/lang/en/components/Dividers.json#L29). TBH, I didn't fully understand the plumbing of this component, but it worked and it seemed to be basically parallel with other section-definition components.

Reply:
That's because `doc/Usage.vue` uses the internalValue for `<doc-example>`

![image](https://user-images.githubusercontent.com/9064066/60405240-84522b80-9b74-11e9-8cc7-e334f2c584ca.png)

Reply:
Ha! Yeah, missed that. Good catch! I updated the PR to remove it.